examples/tutorials/VaraibilityModelExamples.ipynb

```diff
@@ -0,0 +1,302 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The database hosted by the University of Washington (on a machine named 'fatboy'; it seems very likely that I will refer to the database by this name for the rest of this notebook) is sub-divided into tables with each table corresponding to a type of astronomical object (main sequence star, white dwarf, galaxy, etc.).  While some tables represent a union of disparate objects (starsALL is the union of main sequence, RGB, white dwarf, RRLyrae, and blue horizontal branch stars), most of the objects in the database are segregated.  This is relevant, because each class of variable object has its own table.  There are separate tables for Cepheids, RR Lyrae, white dwarfs, and eclipsing binary stars.\n",
+    "\n",
+    "A full list of tables on fatboy can be found here:\n",
+    "\n",
+    "https://confluence.lsstcorp.org/display/SIM/Database+Schema\n",
+    "\n",
+    "Currently, only the tables which are incorporated into starsALL are believed to have a realistic distribution, and even that is only over the LSST survey footprint (no guarantees are made for the northern sky)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will use the table of RR Lyrae as an example to explain how variability works in CatSim."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As described in the example notebook\n",
+    "\n",
+    "`sims_catUtils/examples/tutorials/reading_in_custom_data.ipynb`\n",
+    "\n",
+    "fatboy database tables are accessed through python classes that are daughter classes of `CatalogDBObject`.  Many of these classes are already defined in \n",
+    "\n",
+    "`sims_catUtils/python/lsst/sims/catUtils/baseCatalogModels`\n",
+    "\n",
+    "The class for accessing the table of RR Lyrae can be imported like so:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "from lsst.sims.catUtils.baseCatalogModels import RRLyStarObj"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "rrlyDB = RRLyStarObj()"
+   ]
+  },
+  {
```

Comment:
I'm sure this comes up a lot in the catsim notebooks, but do you usually make a note of the warnings that follow these two cells? In the first you get the "Duplicate object type id" and for the second the usual "geopoint" message.